.dialyzer-ignore.exs

```diff
@@ -0,0 +1,5 @@
+# The list of the files to ignoge could be placed below.
+#   SEE: https://github.com/jeremyjh/dialyxir#elixir-term-format
+[
+  {"lib/kv_behaviour.ex", :unknown_type}
```

Comment:
We should not add entries here, we are supposed to fix issues instead :)

Reply:
I am totally agree, however I made the same empty file for `dialyzer ignore` last time and did not check if it worked. So,  I decided to do it now :D